### PR TITLE
Move email code storage into redis

### PR DESCRIPTION
**Pull request by tomlinton**
Move the email code store into redis and out of session so it works with Brave  etc. I have configured the required env vars for redis.

### CODE REVIEW

**Comment by nick** [comment]
cc @shahthepro as he's done a lot of work on attestations

File: infra/bridge/src/controllers/email.js [hunk 1]
```diff
@@ -50,44 +46,24 @@ router.post('/generate-code', emailGenerateCode, async (req, res) => {
...
 
-  if (req.session.emailAttestation.expiry < new Date()) {
-    logger.warn(
-      `Attempted attestation for ${req.body.email} with expired code.`
-    )
-    return res.status(400).send({
-      errors: ['Verification code has expired.']
-    })
-  }
+  console.log(code)
```
Comment (shahthepro) [inline infra/bridge/src/controllers/email.js]: I guess this log is for debugging. Can be removed.

File: infra/bridge/test/email.js [hunk 2]
```diff
@@ -51,34 +55,17 @@ describe('email attestations', () => {
...
+    const response = await request(app)
       .post('/api/attestations/email/verify')
       .send({
         email: 'origin@protocol.foo',
         code: '123456',
         identity: ethAddress
       })
-      .expect(200)
+
+    // .expect(200)
```
Comment (franckc) [inline infra/bridge/test/email.js]: Why commenting out the check on response code ?

File: infra/bridge/src/controllers/email.js (hunk 1 above)
Comment (tomlinton) [reply infra/bridge/src/controllers/email.js]: Oops, thanks!

File: infra/bridge/test/email.js (hunk 2 above)
Comment (tomlinton) [reply infra/bridge/test/email.js]: Debugging 😬 fixed

**Pull request merged**